### DISCUSSION TITLE
Onkeyup="myFunction()" not rendered in Frontend

### DISCUSSION THREAD

**Discussion by Eiscoffee** [requarks/wiki#4786] [Help / Questions] [answered]
Hello there,

i tried to fix this on my own but i think i can't...

I have an html input like this:
`<input placeholder="myText" id="myInput" type="text"` onkeyup="myFunction()"`  >`

and all i get in the frontend is:

`<input placeholder="myText" id="myInput" type="text">`

If i just change my the html in the console i can write`onkeyup="myFunction()"` in the `<input>` and my function works like i want it to be. But it is obviously not rendered.

Is there any idea how i get this to work?
I also thought of a script to just put `onkeyup="myFunction()"`  afterwards inside the input but my skill are not good enough for sth like this...

Any help is appreciated!

**Comment by NGPixel** [comment, marked as answer]
You need to disable HTML Sanitization from the admin > Rendering > HTML > Security, then save your page again. Because scripts are potentially dangerous, they are stripped by default.

**Reply by Eiscoffee** [reply]
Thank you very much!
